Changelog.md

```diff
@@ -4,19 +4,30 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## Unreleased v1.0.1-SNAPSHOT
```

Comment:
There is no need to add a version number next to "unreleased"

Reply:
To be able to quickly change it once it's release.